### ISSUE TITLE
Backend product list throws deprecation warning and error

### ISSUE THREAD

**Issue by michb** [isotope/core#2130]
**Contao:** `4.9.2` 
**Isotope:** `2.6.10`

When hovering over a row of a product Chrome Devtools shows following warning

```
hover.min.js?v=5c06a00b:1 
The Theme.hoverRow() function has been deprecated in Contao 4 and will be removed in Contao 5. Assign the CSS class "hover-row" instead.
```

and when clicking on a row Chrome Devtools throws following error

```
contao?do=iso_products:340 
Uncaught TypeError: Theme.toggleSelect is not a function
    at HTMLTableRowElement.onclick (contao?do=iso_products:340)
```

This error also occurs if one is in the `edit multiple` mode where you can select the checkbox by clicking anywhere on the row.

**Comment by aschempp** [comment]
That‘s because Isotope still supports Contao 3.5, which requires this line. We should definitely drop/update it in the next release that does no longer support Contao 3.5

**Comment by aschempp** [comment]
Fixed in ebc63604cda0e857cffd05e9498e1a542922d901
